root/static/scripts/release-editor/trackParser.js

```diff
@@ -323,6 +329,13 @@ releaseEditor.trackParser = {
             return data;
         }
 
+        //Use custom delimeter as separator.
+        if (options.useCustomDelimiter && options.customDelimiter != "") {
+          //Escape most regex characters.
```

Comment:
Nit-picking:
* space is missing after comment mark `//`
* indentation is inconsistent with the 4-space indent used in the rest of this file
```suggestion
            // Escape most regex characters.
```